play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukErrorMessage.scala.html

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.hmrcfrontend.views.Utils._
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.errormessage.ErrorMessage
```

Comment:
Are these just added to help IntelliJ?

Reply:
Yes, as on my IDE the twirl template is mostly red and it's difficult to know whats  okay and whats not.